tokens.xml

```diff
@@ -17828,6 +17828,57 @@ If a player casts at least two spells during their own turn, it becomes day next
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>In the Weft</name>
+            <text>After a permanent becomes warped, you can place the exiled card here. You may cast that spell from exile.</text>
+            <prop>
+                <type>State</type>
+                <maintype>State</maintype>
+            </prop>
+            <set picURL="https://github.com/SlightlyCircuitous/image-storage/blob/main/In%20the%20Weft.png?raw=true">EOE</set>
```

Comment:
this is just a redirect to https://raw.githubusercontent.com/SlightlyCircuitous/image-storage/refs/heads/main/In%20the%20Weft.png which one do we use in other tokens?

Reply:
should I change the other links as well?